src/ComponentPage.js

```diff
@@ -50,32 +103,16 @@ class ComponentPage extends Component {
         <TransitionGroup ref={this.initDemoContent} onTransitionEnd={this.handleTransitionEnd_} classes='demo-content-transition'>
           <CSSTransition key={this.props.location.pathname} timeout={350} transitionExitTimeout={0} classNames='loadComponent'>
             <Switch>
-              <Route path='/component/button' component={ButtonCatalog} />
-              <Route path='/component/card' component={CardCatalog} />
-              <Route path='/component/checkbox' component={CheckboxCatalog} />
-              <Route path='/component/chips' component={ChipsCatalog} />
-              <Route path='/component/dialog' component={DialogCatalog} />
-              <Route path='/component/drawer' component={DrawerCatalog} />
-              <Route path='/component/elevation' component={ElevationCatalog} />
-              <Route path='/component/fab' component={FabCatalog} />
-              <Route path='/component/icon-button' component={IconButtonCatalog} />
-              <Route path='/component/image-list' component={ImageListCatalog} />
-              <Route path='/component/layout-grid' component={LayoutGridCatalog} />
-              <Route path='/component/list' component={ListCatalog} />
-              <Route path='/component/linear-progress-indicator' component={LinearProgressIndicatorCatalog} />
-              <Route path='/component/menu' component={MenuCatalog} />
-              <Route path='/component/radio' component={RadioButtonCatalog} />
-              <Route path='/component/ripple' component={RippleCatalog} />
-              <Route path='/component/select' component={SelectCatalog} />
-              <Route path='/component/shape' component={ShapeCatalog} />
-              <Route path='/component/slider' component={SliderCatalog} />
-              <Route path='/component/snackbar' component={SnackbarCatalog} />
-              <Route path='/component/switch' component={SwitchCatalog} />
-              <Route path='/component/tabs' component={TabsCatalog} />
-              <Route path='/component/text-field' component={TextFieldCatalog} />
-              <Route path='/component/theme' component={ThemeCatalog} />
-              <Route path='/component/top-app-bar' component={TopAppBarCatalog} />
-              <Route path='/component/typography' component={TypographyCatalog} />
+              {components.map((component) => {
+                const {filePath, urlPath} = component;
+                const Component = require(`${filePath}`).default;
```

Comment:
This seems like a hack (using node's require within an ES2015 module).

Is it not possible to keep the imports above, then reference them within the array instead of a file path for each one?